### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by HyxonQz**
Altered some strings to be more grammatically accurate and make more sense.

### CODE REVIEW

File: src/instance/minecraft/chat/RepeatChat.ts [hunk 1]
```diff
@@ -6,20 +6,19 @@ import { CommandsManager } from '../CommandsManager'
...
+  "Hold your horses, can't say same message twice!",
+  "Twinkle twinkle little star, can't repeat message with big R",
   'No, no, no, NO. no message repetition D:',
   "RIP, can't say same thing twice",
-  "Wonder where the message have gone? Yeah... can't repeat it :P",
-  'Message can not be repeated!',
+  "Wonder where the message has gone? Yeah... Can't repeat it :P",
+  'Message cannot be repeated!',
   'The verdict has been given and will not be repeated!',
-  "I once tried to say something twice, but then hypixel gave me the L and denied my privilege. Now I can't say the same message twice. Rest in peace my thought.",
```
Comment (aidn3) [inline src/instance/minecraft/chat/RepeatChat.ts:19]: Rest in peace that message. Any reason why completely removing it?

File: src/instance/minecraft/commands/CataCommand.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { Client, SkyblockMember } from 'hypixel-api-reborn'
 const Mojang = require('mojang')
 
 export default {
-  triggers: ['catacomb', 'cata'],
+  triggers: ['catacomb', 'cata', 'c'],
```
Comment (aidn3) [inline src/instance/minecraft/commands/CataCommand.ts:9]: This will conflict with Calculate command.

**Review by aidn3 — CHANGES_REQUESTED** [review]
There are some conflicts.

File: src/instance/minecraft/chat/RepeatChat.ts (hunk 1 above)
Comment (HyxonQz) [reply src/instance/minecraft/chat/RepeatChat.ts:19]: Far too long, would nearly fill the entire chat over 1 error

File: src/instance/minecraft/commands/CataCommand.ts (hunk 2 above)
Comment (HyxonQz) [reply src/instance/minecraft/commands/CataCommand.ts:9]: Whoops, didn't think about that

**Review by aidn3 — APPROVED** [review]
This should resolve it.

**Pull request merged**